test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PrefixedAttributeTagHelpers_Runtime.codegen.cs

```diff
@@ -181,6 +181,10 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_PrefixedAt
             __tagHelperExecutionContext.AddTagHelperAttribute(__tagHelperAttribute_5);
             __TestNamespace_InputTagHelper2.StringDictionaryProperty["paprika"] = (string)__tagHelperAttribute_5.Value;
             __tagHelperExecutionContext.AddTagHelperAttribute(__tagHelperAttribute_5);
+            if (__TestNamespace_InputTagHelper1.StringDictionaryProperty == null)
+            {
+                throw new InvalidOperationException(InvalidTagHelperIndexerAssignment("string-prefix-cumin", "TestNamespace.InputTagHelper1", "StringDictionaryProperty"));
+            }
```

Comment:
@NTaylorMullen - the reason why this is showing up here is that the first usage of this indexer (on line 180) is a preallocated node. 

Do we care enough about this case to fix it?